turbo/execution/eth1/forkchoice.go

```diff
@@ -448,6 +448,15 @@ func (e *EthereumExecutionModule) updateForkChoice(ctx context.Context, original
 				return
 			}
 		}
+
+		// force fsync after notifications are sent
+		if err := e.db.Update(ctx, func(tx kv.RwTx) error {
```

Comment:
Maybe we should add this in ExecV3 when we first sync and have !useExternalTx

Reply:
In this case - no BeginRwNoSync calls.